raiden_installer/installer_parts/raiden_config.py

```diff
@@ -0,0 +1,85 @@
+import os
+import re
+import toml
+from pathlib import Path
+from eth_utils import to_checksum_address
+
+
+class PlainTxtPwd:
```

Comment:
Letters do not cost money, please don't use abbreviations like these - they make it harder to read code, and in the case of more exotic shortenings, promote misunderstandings.